include/mbedtls/arc4.h

```diff
@@ -83,6 +108,11 @@ void mbedtls_arc4_setup( mbedtls_arc4_context *ctx, const unsigned char *key,
  * \param output   buffer for the output data
  *
  * \return         0 if successful
+ *
+ * \warning        ARC4 is considered a weak cipher and its use
+ *                 constitutes a security risk. It is recommended
+ *                 to use a strong cipher instead.
```

Comment:
Should we add a similar warning to the `mbedtls_arc4_self_test()`?

Reply:
I omitted these deliberately, but I don't really care - it's perhaps more uniform to add it there, too.